www-apps/redmine/redmine-3.4.2.ebuild

```diff
@@ -0,0 +1,221 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+# ruby24 dion't have required deps
```

Comment:
I'd say either leave it at 23 and fix afterwards, or at least fix the typo here.